core_backend/tests/api/conftest.py

```diff
@@ -80,38 +83,71 @@ async def asession(
 
 
 @pytest.fixture(scope="session", autouse=True)
-def user(db_session: Session) -> Generator[int, None, None]:
-    global TEST_USER_ID
-    global TEST_USER_ID_2
-    user1_db = UserDB(
-        username=TEST_USERNAME,
-        hashed_password=get_password_salted_hash(TEST_PASSWORD),
-        hashed_api_key=get_key_hash(TEST_USER_API_KEY),
-        content_quota=TEST_CONTENT_QUOTA,
-        created_datetime_utc=datetime.now(timezone.utc),
-        updated_datetime_utc=datetime.now(timezone.utc),
-    )
-    user2_db = UserDB(
-        username=TEST_USERNAME_2,
-        hashed_password=get_password_salted_hash(TEST_PASSWORD_2),
-        hashed_api_key=get_key_hash(TEST_USER_API_KEY_2),
-        content_quota=TEST_CONTENT_QUOTA_2,
-        created_datetime_utc=datetime.now(timezone.utc),
-        updated_datetime_utc=datetime.now(timezone.utc),
+def admin_user(client: TestClient, db_session: Session) -> None:
+    admin_user = UserDB(
+        username=TEST_ADMIN_USERNAME,
+        hashed_password=get_password_salted_hash(TEST_ADMIN_PASSWORD),
+        hashed_api_key=get_key_hash(TEST_ADMIN_API_KEY),
+        content_quota=None,
+        api_daily_quota=None,
+        created_datetime_utc=datetime.utcnow(),
+        updated_datetime_utc=datetime.utcnow(),
     )
-    db_session.add(user1_db)
-    db_session.add(user2_db)
+
+    db_session.add(admin_user)
     db_session.commit()
+    yield admin_user.user_id
```

Comment:
We yield `user_id` for the first one but the second one is lost if we don't save it in a global right?

Reply:
Never mind! I see that you have new fixtures now